easybuild/tools/modules.py

```diff
@@ -118,6 +118,17 @@
 _log = fancylogger.getLogger('modules', fname=False)
 
 
+def module_load_regex(modfilepath):
+    """
+    Return the correct (compiled) regex to extract dependencies, depending on the module file type (Lua vs Tcl)
+    """
+    if modfilepath.endswith('.lua'):
```

Comment:
both the magic suffix and the pattern should come from the generator classes, not hardcoded here.